src/shared_utilities/interpolating_time_varying_inputs.jl

```diff
@@ -2,24 +2,10 @@ InterpolatingTimeVaryingInput =
     Union{InterpolatingTimeVaryingInput0D, InterpolatingTimeVaryingInput2D}
 
 """
-    NearestNeighbor
-
-Return the value corresponding to the point closest to the input time.
-"""
-struct NearestNeighbor <: AbstractInterpolationMethod end
-
-"""
-    LinearInterpolation
-
-Perform linear interpolation between the two neighboring points.
-"""
-struct LinearInterpolation <: AbstractInterpolationMethod end
-
-"""
-    in(time, itp::InterpolatingTimeVaryingInput0D)
+    in(time, itp::InterpolatingTimeVaryingInput)
 
 Check if the given `time` is in the range of definition for `itp`.
 """
-function Base.in(time, itp::InterpolatingTimeVaryingInput0D)
+function Base.in(time, itp::InterpolatingTimeVaryingInput)
```

Comment:
is this still being used? Also, not seeing this "InterpolatingTimeVaryingInput" (without the 0D, 2D suffix)

Reply:
It is. It is defined at the beginning of this file:
```julia
InterpolatingTimeVaryingInput =
    Union{InterpolatingTimeVaryingInput0D, InterpolatingTimeVaryingInput2D}
```